src/shared-components/dialogModal/test.tsx

```diff
@@ -10,13 +8,13 @@ const modalBody = 'Dialog Modal Children Content';
 
 describe('<DialogModal />', () => {
   it('render children content correctly', () => {
-    const wrapper = mount(
+    const { getAllByText } = render(
       <DialogModal title={modalTitle}>
         <div>{modalBody}</div>
       </DialogModal>,
     );
 
-    expect(wrapper.find(ModalTitle).text()).toBe(modalTitle);
-    expect(wrapper.text().includes(modalBody)).toBeTruthy();
+    getAllByText(modalTitle);
```

Comment:
`getBy*` helper methods throw if 0 matches are found, so we don't need to check for truthiness.